databricks/koalas/frame.py

```diff
@@ -1861,7 +1861,8 @@ def get(self, key, default=None):
         except (KeyError, ValueError, IndexError):
             return default
 
-    def sort_values(self, by, ascending=True, inplace=False, na_position='last'):
+    def sort_values(self, by: Union[str, List[str]], ascending: Union[bool, List[bool]] = True,
+                    inplace: bool = False, na_position: str = 'last') -> Union['DataFrame', None]:
```

Comment:
I think the return type can be`Optional['DataFrame']`

Reply:
Fair enough. Fixed it with 862b42f.